images/openshift-kubernetes-nmstate-operator.yml

```diff
@@ -0,0 +1,27 @@
+container_yaml:
+  go:
+    modules:
+      - module: github.com/openshift/kubernetes-nmstate
+content:
+  source:
+    git:
+      branch:
+        target: release-{MAJOR}.{MINOR}
+      url: git@github.com:openshift-priv/kubernetes-nmstate.git
+    dockerfile: build/Dockerfile
+enabled_repos:
+- rhel-server-rpms
+- rhel-server-optional-rpms
+from:
+  builder:
+    - stream: golang
+  member: openshift-enterprise-base
+name: openshift/ose-kubernetes-nmstate-operator
+owners:
+- yboaron@redhat.com
+- asegurap@redhat.com
+- bcrochet@redhat.com
+- bnemec@redhat.com
+- phoracek@redhat.com
+- ellorent@redhat.com
+  
```

Comment:
please provide `for_payload` for new adding images.

Reply:
Thanks for your reminder, please check my update and do you think we should update other yml files as well?

Reply:
@Ximinhan what is holding this up? Is there anything that my team can assist with?

Reply:
1) this patch needs to be rebased
2) streams are crossed, you have rhel7 repos, a (currently) rhel-7 golang builder, and a rhel8 base, that's not going to work (note that we're about to merge the RHEL8 migration which will change much of that, so depending on timeframe, you may need to rework for that anyway)
3) the version of golang may not be changed in streams.yml, that part must be dropped